### DISCUSSION TITLE
Custom Tooltip for apache superset

### DISCUSSION THREAD

**Discussion by NavyaSreekaranam** [apache/superset#26446] [Q&A / Help]
Hi guys, I am a fresher and I need to build a tooltip for the apache superset similar to powerbi where you can get a detailed tooltip chart for main chart.Can anyone help how to do the same?

**Comment by frankzzsword** [comment]
I am also looking for this for our app cause our dashboards have complicated data and sometimes its not clear exactly what it means to our users. Would be cool to add information about the chart they are seeing through a custom tooltip
